tests/testthat/test-get_summary_records.R

```diff
@@ -22,6 +22,46 @@ test_that("get_summary_records Test 1: Summarize average of triplicate ECG inter
     "XYZ-1002", 9,      "QTcF Int. (msec)", "Visit 3",  "2016-03-24T10:56", 402,   "Active 20mg"
   )
 
+  expect_snapshot(
```

Comment:
The title of this test should be updated to "deprecation message if function is called" and the tests below should be renumbered.

Reply:
this is updated